README.md

```diff
@@ -2,55 +2,6 @@
 
 The UID 2 Project is subject to Tech Lab IPR’s Policy and is managed by the IAB Tech Lab Addressability Working Group and Privacy & Rearc Commit Group. Please review [the governance rules](https://github.com/IABTechLab/uid2-core/blob/master/Software%20Development%20and%20Release%20Procedures.md).
 
-This document includes:
-* [Requirements](#requirements)
-* [Install](#install)
-* [Usage](#usage)
-* [Development](#development)
-* [Example Usage](#example-usage)
-
-## Requirements
-
-This SDK supports Python 3.6 and above.
-
-## Install
-
-The SDK can be installed using pip.
-```
-pip install uid2-client
```

Comment:
Do we need to mention how to run the pip in the readme

Reply:
I'll be moving that info to the official guide